lib/bolt/util.rb

```diff
@@ -77,6 +77,15 @@ def read_optional_yaml_hash(path, file_name)
         File.exist?(path) ? read_yaml_hash(path, file_name) : {}
       end
 
+      def first_runs_free
+        path = Bolt::Config.user_path
+        path + '.first_runs_free'
```

Comment:
```suggestion
        Bolt::Config.user_path + '.first_runs_free'
```

Reply:
Oh my gosh I actually only did this because the `noop` bug scared me off :sweat: